Gemfile

```diff
@@ -71,17 +70,17 @@ gem 'logstasher', '0.4.8'
 gem 'browser'
 
 group :test do
-  gem 'rspec-rails', '~> 3.3.3'
-  gem 'capybara', '~> 2.5.0'
+  gem 'rspec-rails', '~> 3.7.1'
+  gem 'rails-controller-testing'
+  gem 'capybara', '~> 2.7.0'
   gem 'capybara-email', '~> 2.4.0'
   gem 'poltergeist', '1.6.0'
-  gem 'database_cleaner', '1.5.0'
+  gem 'database_cleaner', '1.6.2'
```

Comment:
We probably can get rid of this gem. Rails 5.1 introduced system tests that share the same database connection: http://edgeguides.rubyonrails.org/5_1_release_notes.html#system-tests

RSpec 3.7 also supports system tests: http://rspec.info/blog/2017/10/rspec-3-7-has-been-released/

Reply:
I can look into removing that, but potentially worth doing as a separate PR after this one is merged?

Reply:
Sure 👍